{{cookiecutter.project_slug}}/production.yml

```diff
@@ -47,7 +47,7 @@ services:
       {%- endif %}
 
   redis:
-    image: redis:5.0
+    image: redis:6.2.4
```

Comment:
Did you leave the specific version in `local.yml` and `production.yml` on purpose? I don't think we need to be too specific here either.

Reply:
Fixed